crud/src/main/java/com/redhat/lightblue/crud/mongo/MongoCRUDController.java

```diff
@@ -349,11 +347,10 @@ public CRUDFindResponse find(CRUDOperationContext ctx,
                 ctx.addError(Error.get(MongoCrudConstants.ERR_NO_ACCESS, "find:" + ctx.getEntityName()));
             }
         } catch (Error e) {
-            // rethrow lightblue error
-            throw e;
+            ctx.addError(e);
```

Comment:
Why isn't the error logged (LOGGER.error) here? It's very helpful to have all errors logged lightblue side, clients may not log them.